docs: coordinator config (#1959)
spidernet-io/spiderpool: docs/usage/coodinator.md

# Coordinator

> The meta cni-plugin: `coordinator` provides features such as route tuning between pods with multiple NICs, IP address conflict detection, gateway 

Reviewer: ```suggestion
The meta cni-plugin, `coordinator`, provides features such as route tuning between pods with multiple NICs, IP address conflict detection, gateway 
```

Author: Thanks

> reachability detection, and fixed the Mac address of the pod,etc. Coordinator can work in underlay and overlay mode. 

Reviewer: ```suggestion
reachability detection, and fixed the Mac address of the pod, etc. Coordinator can work in underlay and overlay mode. 
```

> - In underlay mode, `coordinator` create a pair of veth device,The one pair is placed to the host, The another one is placed in the net namespace of the pod (called veth0). 

Reviewer: ```suggestion
- In underlay mode, the `coordinator` creates a pair of veth devices. One end of the pair is placed on the host,
  while the other end is placed in the net namespace of the pod, referred to as `veth0`.
```

> this pair of veth devices helps pods forward east-west traffic from the cluster.

Reviewer: ```suggestion
  This pair of veth devices helps pods forward east-west traffic from the cluster.
```

> - In overlay mode, instead of creating a pair of veth device, `coordiantor` forwards east-west traffic to the cluster with the help of the pod's eth0 NIC (often created by some overlay CNI, such as calico or cilium).

Reviewer: ```suggestion
- In overlay mode, rather than creating a pair of veth devices, the `coordinator` forwards east-west traffic to
  the cluster using the pod's eth0 NIC, typically created by an overlay CNI like Calico or Cilium.
```
